script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -1293,61 +1477,63 @@ After all―――
 [sha:68ffba4073c510deaf31a160d5b18f00f47fbe24]{
 -- Page 657, Offset 35153.
 -- 世界が凄い事になっている。
-The world is turning into something dreadful.
+The world was turning into something dreadful.
 }
 [sha:93dc7cfa311767879755e195ccaef04ba622c42c]{
 -- Page 657, Offset 35154.
 -- あってはならない変革が起きている。
-A transformation that must not be is occurring.
+A transformation that should never have occurred was well underway.
 }
 [sha:a6012de25fcd15e89057210fd3e4a0c52d4f61b9]{
 -- Page 657, Offset 35155.
 -- すべてを理解する。
 // well i most certainly dont
-I understand everything that's going on.
+I understood everything.
 }
 [sha:95503091f0f0a3dc7051f814ee853c906b8755e2]{
 -- Page 657, Offset 35156.
 -- 壊れ、幼かった私でさえ進化の目を開かされる。
-Even a broken, young girl like me has been enlightened to evolution.
+Even a young and broken girl like me understood the concept of evolution.
```

Comment:
Uhhh, weird line.
Okay, so since she's kinda describing the ritual, I guess I gotta explain a bit what I think it is.
Remember how Roa wants to fix DAs so they can have kids or something. I think our boy's trying to do some shit like evolve them, soooo...
Btw の目を開かせる is kind of a set-phrase.
>Even a young and broken girl like me had her eyes opened to the evolution.

Reply:
if you think she's referring specifically to Roa's evolution here, I'd suggest capital Evolution, otherwise it looks like an editing mistake

Reply:
Hmmm, maybe italicize it instead. I don't want people thinking its a term.